ast/src/generic.rs

```diff
@@ -624,6 +624,18 @@ impl From<StmtIf> for Ast {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct ElifElseClause {
+    pub range: TextRange,
+    pub test: Option<Expr>,
+    pub body: Vec<Stmt>,
+}
+
+impl Node for ElifElseClause {
+    const NAME: &'static str = "ElifElse";
+    const FIELD_NAMES: &'static [&'static str] = &["test", "body"];
+}
```

Comment:
Unrelated to your change (and this PR): I think we can remove this. It was added for the Pyo3 support which I believe we removed too?